packages/server/src/util/test-utils/seed.ts

```diff
@@ -183,25 +185,36 @@ async function generateOrg(admin: User, numUsers: number, numRosterEntries: numb
 
   // Insert observations per roster entry
   let observations: Observation[] = [];
-  for (let i = 0; i < numRosterEntries; i++) {
-    observations = observations.concat(
-      observationTestData(rosterEntries[i].edipi,
-        rosterEntries[i].unit.name,
-        reportSchemas[0],
-        moment('2020-01-01T08:00:00Z'),
-        moment('2020-01-07T08:00:00Z')),
-    );
-
-    // these observations are for the single-muster config
-    // hence the same timestamp for start and end
-    observations = observations.concat(
-      observationTestData(rosterEntries[i].edipi,
-        rosterEntries[i].unit.name,
-        reportSchemas[1],
-        moment('2020-01-02T10:00:00Z'),
-        moment('2020-01-02T10:00:00Z')),
-    );
-  }
+  const rosterEntriesByUnit = _.groupBy(rosterEntries, entry => entry.unit.name);
+  // for each unit...
+  _.forOwn(rosterEntriesByUnit, (value: Roster[]) => {
+    let count = 0;
+    // for each person on roster for the unit...
+    value.forEach((rosterEntry: Roster) => {
+      // we need to get the unit "number" and unfortunately there is no
+      // elegant way to do this currently aside from parsing it from the name
+      if (count < parseInt(rosterEntry.unit.name.split(' ')[1])) {
```

Comment:
This `count` and the logic is confusing. How does count relate the the numerical part of the unit name? Some magic is happening here. 

I know what this does but only because I am familiar with unit names and some magic used to generated the seed data:
`parseInt(rosterEntry.unit.name.split(' ')[1])`
Maybe wrapping this into a method so that anyone new would not spend too much time trying to understand what is happening here? 

I might be missing something but would it be simpler for each roster (which has edipi & unit name) create two observations? It looks like that is what this code does. It would be a simple forEach loop over `rosterEntries`.

Reply:
The intent of this logic is to limit the number of users submitting observations and ultimately gain more compliance based on higher unit numbers. It does this based on the unit "number" which is equal to `parseInt(rosterEntry.unit.name.split(' ')[1])`. We use this number because IDs can't be trusted to be constant when running tests. <- I'll make sure to add this as a comment.